<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_Text_Browser_Wid</class>
 <widget class="QWidget" name="Sys_Text_Browser_Wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_save">
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/hydro/Save_Document</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_print">
       <property name="text">
        <string>&amp;Print</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/hydro/Print_Document</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/hydro/Remove_Object</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_title">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_path">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/hydroxygen.qrc"/>
 </resources>
 <connections/>
</ui>
